src/main/kotlin/org/rust/cargo/runconfig/CargoRunState.kt

```diff
@@ -49,6 +47,13 @@ class CargoRunState(
         }
     }
 
+    fun cargo(): Cargo = toolchain.cargoOrWrapper(cargoProject?.manifest?.parent)
+
+    fun sysroot(): String? {
```

Comment:
Inconsistent naming: sysroot vs getSysroot. 

I would called this `computeSysroot`, because this does IO, and might fail with ExecutionException.